lib/scope_file_access_flat.rb

```diff
@@ -30,7 +30,11 @@ def binary?(system_file)
     path = system_file.scope.file_path(system_file)
     return false if File.zero?(path)
 
-    output = Cheetah.run("file", path, stdout: :capture)
-    !output.include?("ASCII")
+    f = File.open(path)
+    while (b=f.read(256)) do
```

Comment:
Surrounding space missing for operator '='.<br>Do not use `do` with multi-line `while`.